pvt_cleaning.ipynb

```diff
@@ -0,0 +1,1582 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.        # - get rid of null values
</pre><p>Make sure there is a task for this in notion!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>